src/plugins/intel_gna/src/gna_plugin.cpp

```diff
@@ -663,147 +663,170 @@ void GNAPlugin::AddDebugProperties(const InferenceEngine::CNNLayerPtr layer,
 }
 #endif
 
+static void TransformModel(std::shared_ptr<ngraph::Function> model,
```

Comment:
I would suggest moving the list of this transformation outside of the gna_plugin.cpp

Reply:
Just to simplify gna_plugin.cpp, never mind

Reply:
@mryzhov @kbruniec I would like to keep it here to have both passes registration (legacy and new one) in one file. At least until legacy one will be removed.